addon/components/base/bs-contextual-help.js

```diff
@@ -155,7 +153,7 @@ export default Component.extend(TransitionSupport, {
   transitionDuration: 150,
 
   /**
-   * Keeps the tooltip/popover within the bounds of this element when `autoPlacement` is true. Can be any valid jQuery selector.
+   * Keeps the tooltip/popover within the bounds of this element when `autoPlacement` is true. Can be any valid CSS selector.
```

Comment:
you are right. I hope in most cases it should not matter, but when people use some non-standard jQuery selectors like `: eq` and a bunch of others, this would be a breaking change indeed...